### DISCUSSION TITLE
Helm chart: bucket metrics does not come by default

### DISCUSSION THREAD

**Discussion by norelm** [minio/minio#18852] [Q&A]
Hello, guys 
I am a Devops engineer and I use your helm chart to install minio.
I've noticed a thing missing from your helm chart, I was trying to export metrics to Prometheus using your values file and service monitor.
When metrics enabled, deployment creates a Prometheus probe that exports metrics from the path /minio/v2/metrics/cluster,
also if enabled, it creates a servicemonitor for metrics path /minio/v2/metrics/node. in your documentation, it states that you have another path for bucket metrics - /minio/v2/metrics/bucket, it does work on minio, but helm deployment is missing the promethues probe for it in the servicemonitor.yaml file located in templates. in order for the probe for /bucket to be deployed automatically I had to add it manually to the template file, which will make it hard on future upgrades since I do not relay only on the values file but also the templates folder. 
Am I missing that feature on the helm chart to deploy metrics to prometheus on path /bucket automatically? or is it actually not present on there on  purpose?

Thanks in advance.

**Comment by harshavardhana** [comment]
You need to look at your server version all these metrics are available already.

**Reply by norelm** [reply]
Hey, thanks for your answer,
I will explain my point, the metrics are available on all three paths, there is no issue with them, my point is specific to the helm chart.
when you enable metrics on the values file, it only send metrics for paths /cluster and /node, but not /bucket to prometheus,
I had to go into templates folder into servicemonitor.yaml file, and manually add a probe for path /bucket for helm deployment to automatically send bucket metrics to prometheus when metrics is enabled.
My main point is to know if the probe for /bucket not there on purpose? or I've missed something?

**Reply by harshavardhana** [reply]
For that you need to add support for it, we have no plans to expand on vanilla helm charts. We recommend that you use operator for this.

**Reply by norelm** [reply]
thanks :)

**Comment by norelm** [comment]
got my answer
